types/d3-array/d3-array-tests.ts

```diff
@@ -364,6 +371,70 @@ numOrUndefined = d3Array.scan(readonlyMixedObjectOrUndefinedArray, (a, b) => {
     return a && b ? a.num - b.num : NaN;
 });
 
+// leastIndex() ----------------------------------------------------------------------
+
+numOrUndefined = d3Array.least(numbersArray);
+numOrUndefined = d3Array.least(typedArray);
+numOrUndefined = d3Array.least(readonlyNumbersArray);
+
+let mo: MixedObject | undefined = d3Array.least(mixedObjectArray, (a, b) => {
```

Comment:
Hey @Ledragon, just stumbled upon the PR. A few days ago I started the updates but didn't realize that the PR was already created by you :) Thanks a lot for these updates.

A tiny suggestion if its okay: [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47181/files#diff-e8b9f215cbab373580660b51a4326b3bR50) there are multiple variables which are already declared. Seems like we may not need to redeclare `mo` as `mixedObjectOrUndefined` can do the same task.

Reply:
The hyperlink is supposed to point to:

![Screen Shot 2020-10-03 at 4 10 18 PM](https://user-images.githubusercontent.com/25290212/94989508-fc3aad00-0592-11eb-9271-8fee18b04c98.png)

Reply:
I'll approve this as is, but generally the way to verify types is to use `// $ExpectType`: https://github.com/DefinitelyTyped/DefinitelyTyped/#linter